docs/ui/components/Snippet/blocks/DiffBlock.tsx

```diff
@@ -76,6 +79,15 @@ export const DiffBlock = ({
           operationType={type}
           showOperation={showOperation}
           float={collapseDeletedFiles && type === 'delete'}>
+          {newPath && filenameToLinkUrl && type !== 'delete' ? (
+            <SnippetAction
+              leftSlot={<ArrowUpRightIcon className="text-icon-secondary shrink-0 icon-sm" />}
```

Comment:
```suggestion
              rightSlot={<ArrowUpRightIcon className="text-icon-secondary shrink-0 icon-sm" />}
```

Let's move icon to the right, as we usually do for the arrows, otherwise looks good!